modules/monitoring-contents-of-alertmanager-ui.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// monitoring/configuring-the-monitoring-stack.adoc
+
+[id='contents-of-alertmanager-ui-{context}']
+= Contents of Alertmanager UI
```

Comment:
If we keep this page, the title should be changed since "Alertmanager UI" would normally refer to Alertmanager's own UI. Should it say "Monitoring UI"?